src/Apps/Order/Routes/Status/index.tsx

```diff
@@ -229,6 +229,7 @@ export class StatusRoute extends Component<StatusProps> {
       description,
       showTransactionSummary = true,
     } = this.getStatusCopy()
+    const showOfferNote = order.mode === "OFFER" && order.state === "SUBMITTED"
```

Comment:
at some point (different PR) it might be good to push these strings into an enum

Reply:
I don’t think that would really bring us much more. The TS compiler will already ensure it's one of [the valid values](https://github.com/artsy/reaction/blob/63ac32a0ab0f931f79d8e392c9e0eca405c68972/src/__generated__/Status_order.graphql.ts#L9), so seems like it would then just replace a string with a variable reference?

Reply:
(Ah, if it's already setup to check then that's what i meant.)